src/test/java/com/github/dockerjava/client/AbstractDockerClientTest.java

```diff
@@ -180,13 +183,13 @@ public static void assertContainerHasVolumes(InspectContainerResponse inspectCon
             Volume... expectedVolumes) {
```

Comment:
Thanks for pointing this out. As I'm faced with so many different test frameworks (junit, testng, scalatest, specs2 ....) during my work I never dived into this. Thanks anyway!